<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screen Track &amp; Action</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>470</width>
      <height>120</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="trackCombo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gesture</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Detect Color</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Detect Text</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Detect Update of Region</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mouse Button</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mouse Position</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Trigger:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Action:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="actionCombo">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Shutdown</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Restart</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sleep</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Run a command</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Insert text</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pointer move</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pointer click</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Get Text to Clipboard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Screenshot of Region</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Screen Capture Region</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="regionButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Region</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>471</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hl1"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>211</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hl2"/>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>52</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>52</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
    <widget class="QPushButton" name="doneBtn">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>210</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
